app/src/main/java/wordtextcounter/details/main/feature/notes/NotesViewModel.kt

```diff
@@ -0,0 +1,14 @@
+package wordtextcounter.details.main.feature.notes
+
+import wordtextcounter.details.main.feature.base.BaseViewModel
+
+/**
+ * Created by hirak on 13/03/18.
+ */
```

Comment:
I'm not sure we should include authors at the top of the files. I removed all yesterday. Let's not include these tags.

Reply:
Will do